CHANGES.rst

```diff
@@ -127,6 +127,10 @@ API Changes
   - Angstrom, erg, G, and barn are no more reported as deprecated FITS units.
     [#5929]
 
+  - ``comments`` meta key is output to ``COMMENT`` instead of ``COMMENTS``
```

Comment:
Maybe useful to add that the "`comments` meta key" refers the `io.ascii / table` convention.